flow/layers/backdrop_filter_layer.cc

```diff
@@ -60,15 +60,13 @@ void BackdropFilterLayer::Paint(PaintContext& context) const {
   AutoCachePaint save_paint(context);
   save_paint.setBlendMode(blend_mode_);
   if (context.leaf_nodes_builder) {
-    context.leaf_nodes_builder->saveLayer(&paint_bounds(),
-                                          save_paint.dl_paint(), filter_.get());
-
+    auto save_layer = Layer::DlAutoSaveLayer::Create(
```

Comment:
Let's get rid of the new DlAutoSaveLayer here. It is raising too many questions outside the scope of converting IFLayer to DlImageFilter and it really should be a separate PR.

Reply:
recovered